content/hardware/06.nicla/boards/nicla-voice/tutorials/getting-started-ml/content.md

```diff
@@ -1,160 +1,196 @@
-
-### Installing Additional Dependencies
-
-To easily flash any Machine Learning model created with Machine Learning Tools integration onto the Nicla Voice, we need to install the Arduino CLI and Edge Impulse® CLI. Follow [this article](https://docs.edgeimpulse.com/docs/development-platforms/officially-supported-mcu-targets/arduino-nicla-vision#installing-dependencies) from Edge Impulse® to install both properly.
-
-### Uploading the Model
-
-Now that everything needed for flashing the firmware and the model to the Nicla Voice is installed, you can finally flash the board with your model. Extract the files that were packed into the .zip file received from Edge Impulse® when the model was built into a folder. At this point, run the right "flash" file for your OS on the machine you are using, as reported in this list:
-
-- Use **flash_windows.bat** if you are using a PC
-- Use **flash_mac.command** if you are using a MAC
-- Use **flash_linux.sh** if you are using a Linux machine
-
-If there is any issue in the process, try to run these as administrator. A command window will appear where you can follow the status of the installation.
-
-### Testing It Out
-
-To make sure everything is working open the Arduino IDE, select the correct port that the board is connected to, and open the serial monitor. If there is nothing in the serial monitor try pressing the reset button on the Nicla Voice once. The information about the files and the model that is loaded onto the board should now be printed on the serial monitor.
-
-![Correct firmware and model loaded on the Nicla Voice](assets/fw-loaded-serial-monitor.svg)
-
-Now you can go ahead and say your wake up words to the Nicla Voice. The board should then blink the blue LED when a sound is registered. If the LED does not blink try moving closer to the Nicla Voice or pronounce the wake up words differently. The board will print what class it thinks the sounds recognized belongs to, as seen in the image below. This makes it easy to see if the board will recognize the desired phrase or if it is recognized as a different class. If the board has problems recognizing the right phrase it might be a good idea to take another look at the model and the data that has been collected.
-
-![Serial monitor showing sound recognized](assets/phrase-recognition.svg)
-
-## Conclusion
-
-This tutorial showed how to set up the Machine Learning Tools platform to create a machine learning model to be used with the Nicla Voice. It went through how to collect audio with a phone, how to train the model with the data, and then how to build it to work with the board. Lastly, the tutorial shows how to upload the model to the Nicla Voice and how to use that model.
+---
+title: 'Getting Started with Speech Recognition and the Nicla Voice'

```

Comment:
I would suggest: "Getting started with speech/sound recognition on the Nicla Voice". 
I'm just worried that with this current title people get confused with the [Arduino Speech Recognition Engine](https://docs.arduino.cc/tutorials/portenta-vision-shield/speech-recognition-engine), which it's not related to Nicla Voice. What do you think @canchebagur?

Reply:
I like:
* "Getting started with sound recognition on the Nicla Voice"
* "Getting started with Machine Learning audio models"

If we avoid the words "speech" and "recognition" we should be fine

Reply:
We will go with the second one:

Getting started with Machine Learning audio models on the Nicla Voice